docs/src/pages/tutorial/run-in-docker.md

```diff
@@ -25,3 +32,32 @@ Or you can stop the container with
 ```bash
 docker stop monika
 ```
+
+For more complex containers, you can pass it monika parameters. Using prometheus with monika in a docker, you would pass the parameters like so:
+
+```bash
+docker run --name monika_interactive \
+    -v ${PWD}/myConfig.yml:/config/monika.yml \
+    -d hyperjump/monika:latest \
+    monika -c /config/monika.yml --prometheus 3001
+```
+
+## Troubleshooting
+
+Genererally when facing issues with your container, try the parameters using regular monika. For instance:
+
+```bash
+monika -c /config/monika.yml --prometheus 3001
+```
+
+Ensure your container is up and running by running simple commands/parameters:
+
+```bash
+docker run --name monika_interactive \
+    -v ${PWD}/myConfig.yml:/config/monika.yml \
+    -d hyperjump/monika:latest monika --help
+```
+
+For further docker commands and documentation, visit the official Docker [documentation here](https://docs.docker.com/engine/reference/commandline/run/).
+
+If all else fails, hit us up at https://github.com/hyperjumptech/monika.
```

Comment:
maybe put direct link to [monika discussions](https://github.com/hyperjumptech/monika/discussions) or [file a bug report](https://github.com/hyperjumptech/monika/issues)?

Reply:
good tip. thanks.